primitives/src/script/mod.rs

```diff
@@ -542,6 +542,7 @@ impl<'de> serde::Deserialize<'de> for &'de Script {
 #[cfg(feature = "serde")]
 impl serde::Serialize for ScriptBuf {
     /// User-facing serialization for `Script`.
+    #[inline]
     fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
I forgot to mention that `#[inline]` on generic functions is not needed.

Reply:
Do you want me to remove this even though you have approved the PR?

Reply:
You may, I'll be able to approve quickly.

Reply:
I have removed inline from the generic functions